src/ui/public/management/section.js

```diff
@@ -20,10 +23,18 @@ export default class ManagementSection {
       index: ['id'],
       order: ['order']
     });
+    this.visible = true;
+    this.disabled = false;
+    this.tooltip = '';
+    this.url = '';
 
     assign(this, options);
   }
 
+  get visibleItems() {
+    return this.items.inOrder.filter(item => item.visible);
```

Comment:
Since `items` is an `IndexedArray`, couldn't we specify _visible_ and _disabled_ as a group and use `byVisible`? Thoughts?